### PR TITLE
bitnami version

### PR DESCRIPTION
**Pull request by chamer81**
This version simply replaces the postgres image with the bitnami version to test if it will work better and be more robust.

### CODE REVIEW

File: {{cookiecutter.project_slug}}/_/deployment/database/base/deployment.yaml [hunk 1]
```diff
@@ -10,7 +10,7 @@ spec:
     spec:
       containers:
       - name: postgres
-        image: docker-registry.default.svc:5000/openshift/postgresql:9.6
+        image: docker.io/bitnami/postgresql:9.6
```
Comment (bittner) [inline {{cookiecutter.project_slug}}/_/deployment/database/base/deployment.yaml]: Is there a specific reason why we stick to version 9.6? Could we move on to version 12 instead? There are a few worthwhile new features in Postgres version>10.

### References

- [PostgreSQL 10 release notes](https://www.postgresql.org/docs/10/release-10.html)
- [PostgreSQL 11 release notes](https://www.postgresql.org/docs/11/release-11.html)
- [PostgreSQL 12 release notes](https://www.postgresql.org/docs/12/release-12.html)
- [Postgres versioning policy](https://www.postgresql.org/support/versioning/)

File: {{cookiecutter.project_slug}}/_/deployment/database/base/deployment.yaml [hunk 2]
```diff
@@ -28,7 +28,8 @@ spec:
             - /bin/sh
             - -i
             - -c
-            - psql -h 127.0.0.1 -U $POSTGRESQL_USER -q -d $POSTGRESQL_DATABASE -c
+            - PGPASSWORD=$POSTGRESQL_PASSWORD 
+              psql -h 127.0.0.1 -U $POSTGRESQL_USERNAME -q -d $POSTGRESQL_DATABASE -c
```
Comment (bittner) [inline {{cookiecutter.project_slug}}/_/deployment/database/base/deployment.yaml]: I don't understand to role of `PGPASSWORD`. There is no reference to it [in the image documentation](https://hub.docker.com/r/bitnami/postgresql) on Docker Hub.

File: {{cookiecutter.project_slug}}/_/ci-services/definitions/.gitlab-ci.yml [hunk 3]
```diff
@@ -49,14 +49,15 @@ stages:
   before_script:
   - export DJANGO_SECRET_KEY=$(cat /dev/urandom | tr -dc A-Za-z0-9 | head -c50)
   - export POSTGRES_PASSWORD=$(cat /dev/urandom | tr -dc A-Za-z0-9 | head -c16)
+  - export POSTGRESQL_USERNAME={{ cookiecutter.project_slug.replace('-', '_') }}
```
Comment (bittner) [inline {{cookiecutter.project_slug}}/_/ci-services/definitions/.gitlab-ci.yml]: It seems to me, we should align the variable names here to reduce the distraction.

File: {{cookiecutter.project_slug}}/_/deployment/database/base/deployment.yaml (hunk 1 above)
Comment (chamer81) [reply {{cookiecutter.project_slug}}/_/deployment/database/base/deployment.yaml]: No, no reason.  I just used the version that was already in the file.  Yes, we should try out updating the version.

File: {{cookiecutter.project_slug}}/_/deployment/database/base/deployment.yaml (hunk 2 above)
Comment (chamer81) [reply {{cookiecutter.project_slug}}/_/deployment/database/base/deployment.yaml]: This was an underdocumented feature that we found when troubleshooting (with @srueg ).  The bitnami postgres image reads the password from this environment variable and there's no variant to put the password on the command-line.  So, either we set it in the same command, or it waits forever for the password.

File: {{cookiecutter.project_slug}}/_/ci-services/definitions/.gitlab-ci.yml (hunk 3 above)
Comment (chamer81) [reply {{cookiecutter.project_slug}}/_/ci-services/definitions/.gitlab-ci.yml]: Yes, good point, that would be cleaner.

**Pull request merged**